ami/trainers/forward_dynamics_trainer.py

```diff
@@ -0,0 +1,75 @@
+from functools import partial
+
+import torch
+from torch.distributions import kl_divergence
+from torch.distributions.normal import Normal
+from torch.nn.functional import mse_loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from ami.data.buffers.buffer_names import BufferNames
+from ami.data.buffers.causal_data_buffer import CausalDataBuffer
+from ami.data.buffers.random_data_buffer import RandomDataBuffer
+from ami.data.interfaces import ThreadSafeDataUser
+from ami.models.components.forward_dynamics import ForwardDynamics
+from ami.models.model_names import ModelNames
+from ami.models.model_wrapper import ModelWrapper
+from ami.models.vae import VAE, Decoder, Encoder
+
+from .base_trainer import BaseTrainer
+
+
+class ForwardDynamicsTrainer(BaseTrainer):
+    def __init__(
+        self,
+        partial_dataloader: partial[DataLoader[torch.Tensor]],
+        partial_optimizer: partial[Optimizer],
+        device: torch.device,
+    ) -> None:
```

Comment:
epoch数と学習可能な最小データサイズを指定可能にしてください。
参考:https://github.com/MLShukai/ami/blob/main/ami/trainers/ppo_policy_trainer.py#L39-L40
```suggestion
    def __init__(
        self,
        partial_dataloader: partial[DataLoader[torch.Tensor]],
        partial_optimizer: partial[Optimizer],
        device: torch.device,
        max_epochs: int = 1,
        minimum_dataset_size: int = 1,
    ) -> None:
```

Reply:
対応しました https://github.com/MLShukai/ami/pull/123/commits/9aa4e1a64132a065599a5c5f253853e19b87d20f